src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final double AREA_COEFFICIENT = 0.5;
+    private final double smallBase;
+    private final double bigBase;
+    private final double height;
+    private final double leg;
+
+    public IsoscelesTrapezoid(String colour, double smallBase, double bigBase, double height) {
+        super(colour);
+        this.smallBase = super.roundValues(smallBase);
+        this.bigBase = super.roundValues(bigBase);
+        this.height = super.roundValues(height);
+        this.leg = countLeg();
```

Comment:
And let's not do it inside the constructor. This is kind of unexpected behavior. For example, you can do this in a draw method.

Reply:
The point was not to provide set method for each variable, but to create a method which makes a proper value for the field